services/QuillLMS/db/migrate/20241121214425_add_feedback_history_index_on_several_columns.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class AddFeedbackHistoryIndexOnSeveralColumns < ActiveRecord::Migration[7.1]
+  # An index can be created concurrently only outside of a transaction.
+  disable_ddl_transaction!
```

Comment:
Will running this outside of a transaction cause issues with pgBouncer?  If there are multiple commands, pgBouncer will only run them on the same connection for sure if they're in a transaction.  This might just be a single command, though, so maybe it's not a risk.

Reply:
It's just a single command. Always good to think about pbBouncer though

Reply:
* or maybe you were implying that CREATE INDEX can fork off additional commands? I don't think so / hope not :-)